website/docs/r/slb_acl.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_slb_acl"
+sidebar_current: "docs-alicloud-resource-slb-acl"
+description: |-
+  Provides a Load Banlancer Access Control List resource.
+---
+
+# alicloud\_slb\_acl
+
+A access control list contains multiple IP addresses or CIDR blocks.
+The access control list can help you to define multiple instance listening dimension,
+and to meet the multiple usage for single access control list.
+
+Server Load Balancer allows you to configure access control for listeners.
+You can configure different whitelists or blacklists for different listeners.
+
+You can configure access control
+when you create a listener or change access control configuration after a listener is created.
+
+~> **NOTE:** One access control list can be attached with multiple Listener in different load balancer as whitelists or blacklists.
+~> **NOTE:** The maximum number of access control lists per region  is 50.
+~> **NOTE:** The maximum number of IP addresses added each time is 50.
+~> **NOTE:** The maximum number of entries per access control list is 300.
+~> **NOTE:** The maximum number of listeners that an access control list can be added to is 50.
+
+## Example Usage
+
+```
+   # Create a new access control list
+
+   resource "alicloud_slb_acl" "foo" {
+     name = "tf-testAccSlbAcl"
+     ip_version = "ipv4"
+     entrys = [
+       {
+         entry="10.10.10.0/24"
+         comment="first-a"
+       },
+       {
+         entry="168.10.10.0/24"
+         comment="abc-test-abc-b"
+       },
+     ]
+   }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `name` - (Required) Name of the access control list.
+* `ip_version` - (Optional, ForceNew) The IP Version of access control list is the type of its entry (IP addresses or CIDR blocks). It values ipv4/ipv6. Our plugin provides a default ip_version: "ipv4".
+* `entrys` - (Optional) A list of entry (IP addresses or CIDR blocks) to be added. At most 50 etnry can be supported in one resource. It contains two sub-fields as `Entry Block` follows.
+
+## Entry Block
+
+The entry mapping supports the following:
+* `entry` - (Required) An IP addresses or CIDR blocks.
+* `comment` - (Optional) the comment of the entry.
+
+## Attributes Reference
+
+The following attributes are exported:
+* `name` - The Name of the access control list.
```

Comment:
Please add a blank line between line 63 and 64.

In addition, there are no need to show `name`, `ip_version` and `entrys` because of they are input parameter and default to a output parameter.

Please add a new output parameter `id`.